base/ltfssaxes.dtx

```diff
@@ -192,503 +248,3158 @@
+\DeclareFontSeriesChangeRule {ebsc}{c}  {ebc}  {}
+\DeclareFontSeriesChangeRule {ebsc}{sc} {ebsc} {}
+\DeclareFontSeriesChangeRule {ebsc}{sx} {ebsx} {}
+\DeclareFontSeriesChangeRule {ebsc}{x}  {ebx}  {}
+\DeclareFontSeriesChangeRule {ebsc}{ex} {ebex} {}
+\DeclareFontSeriesChangeRule {ebsc}{ux} {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {eb}{uc}   {ebuc} {}
+\DeclareFontSeriesChangeRule {eb}{ec}   {ebec} {}
+\DeclareFontSeriesChangeRule {eb}{c}    {ebc}  {}
+\DeclareFontSeriesChangeRule {eb}{sc}   {ebsc} {}
+\DeclareFontSeriesChangeRule {eb}{sx}   {ebsx} {}
+\DeclareFontSeriesChangeRule {eb}{x}    {ebx}  {}
+\DeclareFontSeriesChangeRule {eb}{ex}   {ebex} {}
+\DeclareFontSeriesChangeRule {eb}{ux}   {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebsx}{ul} {ulsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{el} {elsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{l}  {lsx}  {}
+\DeclareFontSeriesChangeRule {ebsx}{sl} {slsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{sb} {sbsx} {bsx}
+\DeclareFontSeriesChangeRule {ebsx}{b}  {bsx}  {}
+\DeclareFontSeriesChangeRule {ebsx}{eb} {ebsx} {bsx}
+\DeclareFontSeriesChangeRule {ebsx}{ub} {ubsx} {bsx}
+\DeclareFontSeriesChangeRule {ebsx}{uc} {ebuc} {}
+\DeclareFontSeriesChangeRule {ebsx}{ec} {ebec} {}
+\DeclareFontSeriesChangeRule {ebsx}{c}  {ebc}  {}
+\DeclareFontSeriesChangeRule {ebsx}{sc} {ebsc} {}
+\DeclareFontSeriesChangeRule {ebsx}{sx} {ebsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{x}  {ebx}  {}
+\DeclareFontSeriesChangeRule {ebsx}{ex} {ebex} {}
+\DeclareFontSeriesChangeRule {ebsx}{ux} {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebx}{ul}  {ulx}  {}
+\DeclareFontSeriesChangeRule {ebx}{el}  {elx}  {}
+\DeclareFontSeriesChangeRule {ebx}{l}   {lx}   {}
+\DeclareFontSeriesChangeRule {ebx}{sl}  {slx}  {}
+\DeclareFontSeriesChangeRule {ebx}{sb}  {sbx}  {bx}
+\DeclareFontSeriesChangeRule {ebx}{b}   {bx}   {}
+\DeclareFontSeriesChangeRule {ebx}{eb}  {ebx}  {bx}
+\DeclareFontSeriesChangeRule {ebx}{ub}  {ubx}  {bx}
+\DeclareFontSeriesChangeRule {ebx}{uc}  {ebuc} {}
+\DeclareFontSeriesChangeRule {ebx}{ec}  {ebec} {}
+\DeclareFontSeriesChangeRule {ebx}{c}   {ebc}  {}
+\DeclareFontSeriesChangeRule {ebx}{sc}  {ebsc} {}
+\DeclareFontSeriesChangeRule {ebx}{sx}  {ebsx} {}
+\DeclareFontSeriesChangeRule {ebx}{x}   {ebx}  {}
+\DeclareFontSeriesChangeRule {ebx}{ex}  {ebex} {}
+\DeclareFontSeriesChangeRule {ebx}{ux}  {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebex}{ul} {ulex} {}
+\DeclareFontSeriesChangeRule {ebex}{el} {elex} {}
+\DeclareFontSeriesChangeRule {ebex}{l}  {lex}  {}
+\DeclareFontSeriesChangeRule {ebex}{sl} {slex} {}
+\DeclareFontSeriesChangeRule {ebex}{sb} {sbex} {bex}
+\DeclareFontSeriesChangeRule {ebex}{b}  {bex}  {}
+\DeclareFontSeriesChangeRule {ebex}{eb} {ebex} {}
+\DeclareFontSeriesChangeRule {ebex}{ub} {ubex} {bex}
```

Comment:
```suggestion
\DeclareFontSeriesChangeRule {ebex}{ub} {ubex} {ebex}
```

I think it is better to use `eb` instead of `b` in the fourth argument: `eb` is a better approximation for `ub` and since the fourth argument is then identical to the first, we can assume that this font face actually exists.